modules/govuk/manifests/apps/email_alert_frontend.pp

```diff
@@ -41,6 +45,11 @@
     app => 'email-alert-frontend',
   }
 
+  govuk::app::envvar { "${title}-PUBLISHING_API_BEARER_TOKEN":
+    varname => 'PUBLISHING_API_BEARER_TOKEN',
+    value   => $publishing_api_bearer_token,
```

Comment:
Need a semicolon at the end of this line to end the statement - you can also combine this with the `envvar` declaration below for conciseness